wayland-backend/src/client_api.rs

```diff
@@ -48,6 +50,11 @@ impl std::fmt::Debug for dyn ObjectData {
 downcast_rs::impl_downcast!(sync ObjectData);
 
 /// An ID representing a Wayland object
+///
+/// The backend internally tracks which IDs are still valid, invalidates them when the protocol object they
+/// represent is destroyed. As such even though the Wayland protocol reuses IDs, you still confidently compare
```

Comment:
Not sure, but was this intended to read "you **can** confidently compare" or is the sentence correct this way? :thinking: